dev/buildtool/spin_commands.py

```diff
@@ -192,17 +194,15 @@ def __init__(self, factory, options, **kwargs):
     self.__scm = BranchSourceCodeManager(options_copy, self.get_input_dir())
     # TODO(jacobkiefer): Add spin CLI autogenerated docs.
 
-    dash = spinnaker_version.find('-')
-    semver_str = spinnaker_version[0:dash]
-    semver_parts = semver_str.split('.')
+    semver_parts = spinnaker_version.split('.')
     if len(semver_parts) != 3:
       raise_and_log_error(
           ConfigError('Expected spinnaker version in the form X.Y.Z-N'))
     self.__release_branch = 'release-{maj}.{min}.x'.format(
         maj=semver_parts[0], min=semver_parts[1])
-    self.__release_tag = 'version-' + semver_str
+    self.__release_tag = 'version-' + spinnaker_version
 
-    self.__release_version = semver_str
+    self.__release_version = spinnaker_version
```

Comment:
So you cannot patch spin without a full spinnaker release?

Reply:
That was unused. Removed.

Reply:
But this still raises a good point about fixes in spin.